### ISSUE TITLE
Bug: Static image loading

### ISSUE THREAD

**Issue by dmgolembiowski** [dmgolembiowski/chess#6]
The call to `Image::load_image_from_mem` reports an empty buffer for data loaded in from `include_bytes!` during testing. During these same tests, the call does work for loading image files dynamically (`Image::load_image`).

More investigative work is needed to verify whether:
- An existing raylib thread needs to be created first
- Weirdness involving `#[test]`, `static`, and `include_bytes!`
- Is the image being decoded (during `load_image` from file) before set into an underlying buffer that we can dump?
- Can it use late initialization using lazy-static?

**Comment by dmgolembiowski** [comment]
It seems possible that my `const` loading of the image might have been correct (when I had used `include_bytes!`). It seems `Image` data needs to be displayed after casting it to be a `Texture2D`.
https://www.reddit.com/r/raylib/comments/17oyly2/2d_image_vs_texture_usage/

**Comment by dmgolembiowski** [comment]
Furthermore, part of the reason why my call to `.unwrap()` caused a test case to fail could be attributed to some new information I discovered about how the screen needs to first be initialized:

```

    // NOTE: Textures MUST be loaded after Window initialization (OpenGL context is required)

    Image image = LoadImage("resources/raylib_logo.png");     // Loaded in CPU memory (RAM)
    Texture2D texture = LoadTextureFromImage(image);          // Image converted to texture, GPU memory (VRAM)
    UnloadImage(image);   // Once image has been converted to texture and uploaded to VRAM, it can be unloaded from RAM
```

**Comment by dmgolembiowski** [comment]
This bug was fixed in 0668be6 where `img_data.rs` was introduced in chess-ray
